spec/integration/use_cases/accounts/apps/evolution_api/webhooks/evolution_apis_controller_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+require 'webmock/rspec'
+require 'sidekiq/testing'
+
+RSpec.describe Apps::EvolutionApisController, type: :request do
+  let(:account) { create(:account) }
+  let(:delete_instance_response) do
+    File.read('spec/integration/use_cases/accounts/apps/evolution_api/instance/delete_response.json')
+  end
+
+  def qrcode_updated_webhook_event_params(evolution_api)
+    {
+      "event": 'qrcode.updated',
+      "instance": evolution_api.instance,
+      "data": {
+        "qrcode": {
+          "instance": evolution_api.instance,
+          "pairingCode": 'V37R3T7X',
+          "code": '2@xCLTLQpXqmnR1jld2wKbVGmXchdVSFR3QHQa+6BNF8EK8VeoKUTYklFySHt0rrgknAPgwBkD7y1IIQ==,Xl787d38Gmn0wbPMtlvd47VIqs1xzfsZQTTLgePiYmA=,XQTEs9Fx080JHolGhWrlheRKQo8WCPj0DqCbrrMLiU8=,r8hWahUUJqu7BJFq1l1dQopW2r3lEbOrWOyQJfmmZdY=',
+          "base64": 'qrcode'
+        }
+      },
+      "destination": 'https://webhook.com/',
+      "date_time": '2024-01-27T02:35:43.230Z',
+      "server_url": 'https://server.com',
+      "apikey": evolution_api.token
+    }
+  end
+
+  def connection_event_params(evolution_api, status_reason, status = 'open')
+    {
+      "event": 'connection.update',
+      "instance": evolution_api.instance,
+      "data": {
+        "instance": evolution_api.instance,
+        "state": status,
+        "statusReason": status_reason
+      },
+      "destination": 'https://webhook.com/',
+      "date_time": '2024-01-27T01:19:47.979Z',
+      "sender": '5522999999999@s.whatsapp.net',
+      "server_url": 'https://app-beta.woofedcrm.com/',
+      "apikey": evolution_api.token
+    }
+  end
+
+  def post_webhook(event_data)
+    Sidekiq::Testing.inline! { post '/apps/evolution_apis/webhooks', params: event_data }
+  end
+
+  def expect_success
+    expect(response).to have_http_status(200)
+    expect(JSON.parse(response.body)).to eq({ 'ok' => true })
+  end
+
+  describe 'POST /apps/evolution_apis/webhooks' do
+    describe 'when evolution_api is connecting' do
+      let!(:evolution_api_connecting) { create(:apps_evolution_api, :connecting, account: account) }
+      context 'when is qrcode_update event ' do
+        it 'should update qrcode' do
+          expect do
+            post_webhook(qrcode_updated_webhook_event_params(evolution_api_connecting))
+          end.to change { evolution_api_connecting.reload.qrcode }
```

Comment:
```suggestion
          end.to change { evolution_api_connecting.qrcode }
```

Aqui sabe se precisa ter o reload?

Reply:
Precisa mesmo